perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -2110,7 +2110,7 @@ public void deleteAttribute(PerunSession sess, AttributeDefinition attribute) th
 
 			jdbc.update("DELETE FROM "+ attributeToTablePrefix(attribute)+"_attr_values WHERE attr_id=?", attribute.getId());
 			jdbc.update("DELETE FROM attr_names WHERE id=?", attribute.getId());
-			log.info("Attribute deleted [{}]", attribute);
+			log.debug("Attribute deleted [{}]", attribute);
```

Comment:
Attribute object is unncessary enclosed in ``[ ]``, opposite to createAttribute() method/message. I suggest ``Attribute created: {}`` and ``Attribute deleted: {}``.